### PR TITLE
Include a warning message when abort is initiated

### PR DESCRIPTION
**Pull request by ramkrsna**
Make sure we get confirmation from admin/user before proceeding with abort.

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/132)

### CODE REVIEW

**Comment by ramkrsna** [comment]
@MohamedAshiqrh @raghavendra-talur @humblec Please take a look and suggest changes.

**Comment by humblec** [comment]
LGTM.

File: deploy/gk-deploy [hunk 1]
```diff
@@ -386,9 +386,18 @@ else
...
+  case $abortopt in
+    Y*|y*)
+    abort
+    ;;
+    N*|n*)
+    exit
+  esac
 fi
 
+
```
Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: Remove this line.

**Comment by humblec** [comment]
Updated the description of the PR . @ramkrsna please address @MohamedAshiqrh's comment. I will merge it once its done.

**Pull request merged**